### DISCUSSION TITLE
Need help getting set up with this action

### DISCUSSION THREAD

**Discussion by raiguard** [crowdin/github-action#69] [Q&A] [answered]
I'm opening a discussion instead of an issue because this is almost certainly user error instead of a problem with the action itself.

I recently took ownership of a fairly mature Factorio mod, and I migrated its source code to GitHub as part of this. The mod has a translation project on Crowdin so I set out to automate the process of updating the translations and such (the previous owner just did it manually).

Anyway, the problem is similar to #60, in that the message is the same, but it happens every time instead of randomly:
```
⚠️  Downloaded translations don't match the current project configuration. The translations for the following sources will be omitted (use --verbose to get the list of the omitted translations):
	- Krastorio2.cfg (1)
	- Krastorio2Compatibility.cfg (8)
	- Krastorio2Wiki.cfg (17)
	- Krastorio2Tips.cfg (1)
```

**Links:**
[The failing action](https://github.com/raiguard/krastorio-2/runs/2561927841?check_suite_focus=true#step:4:29)
[Crowdin project](https://crowdin.com/project/krastorio-2)
[.crowdin.yml](https://github.com/raiguard/krastorio-2/blob/master/.crowdin.yml)
[workflows/crowdin.yml](https://github.com/raiguard/krastorio-2/blob/master/.github/workflows/crowdin.yml)

The sources are in `/locale/en/*.cfg`, and the translations need to be downloaded to `/locale/%locale%/*.cfg`. For testing purposes, I only have the action set to download translations at the moment, and I have it set to only run when I click the button.

Thanks in advance for the help, and sorry for my inexperience.

**Comment by crowdin-support** [comment]
Hi there, sorry for the long reply, we're looking into that already!

**Comment by crowdin-support** [comment, marked as answer]
Hi, as promised I'm coming with the updates!
It requires to change the **Resulting file after translation export** in the file settings to where you would need the translations to be exported, for example: `/locale/%locale%/Krastorio2Compatibility.cfg`
![export](https://user-images.githubusercontent.com/49648514/117978541-a86b2000-b33a-11eb-9bb8-43db9fca4ca3.png)

**Reply by raiguard** [reply]
Thank you! I knew it would be a simple solution.

**Reply by raiguard** [reply]
Disregard, I discovered the language mapping feature, which solves this problem.

~~Ok, as often happens, there are more problems to solve. Factorio uses the two-letter identifier for most languages, and then the full identifier for some others:~~
![image](https://user-images.githubusercontent.com/3515394/118025568-39cb9980-b31d-11eb-8b63-71f1530d490a.png)
~~Unfortunately, using `%locale%` will always use the full identifiers, and therefore the game will not accept most of the translations. I can't change the setting to use two letter identifiers because some of them need the full identifier. How can I handle this?~~
